CHANGES.md

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.374
+- Update model version v0.0.319
```

Comment:
@machi1990 it's 0.0.326

Reply:
Good catch.  I just realised now that I pushed an old branch.